e2e/benchmarks/local-benchmark/index.html

```diff
@@ -110,11 +103,70 @@ <h2>TensorFlow.js Model Benchmark</h2>
     let warmupTimes = 1;
     // Default do not run any task.
     let task = '';
+
+    function loadScript(url, callback) {
+      var script = document.createElement("script")
+      script.type = "text/javascript";
+
+      if (script.readyState) { //IE
+        script.onreadystatechange = function() {
+          if (script.readyState == "loaded" ||
+          script.readyState == "complete") {
+            script.onreadystatechange = null;
+            callback();
+          }
+        };
+      } else { //Others
+        script.onload = function() {
+          callback();
+        };
+      }
+
+      script.src = url;
+      document.getElementsByTagName("head")[0].appendChild(script);
+    }
+
+    var unpkgUrlsIndex = 0;
+    var unpkgUrls = ["https://unpkg.com/@tensorflow/tfjs-core@latest/dist/tf-core.js",
+      "https://unpkg.com/@tensorflow/tfjs-backend-cpu@latest/dist/tf-backend-cpu.js",
```

Comment:
We can generate the link more dynamically. Don't hard code "https://unpkg.com/@tensorflow", “@latest” and so on again and again.